contribute/en/Release-Process.md

```diff
@@ -284,10 +284,10 @@ Current state: <!-- Planning|Translations (beta)|Code freeze (rc)|Released -->
 - [ ] [Tag a release candidate](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release) (inform emlynmac for signing on macOS and upload signed binary from his repo to ours).
   - [ ] Announce the release candidate on Github Discussions. Pin the thread. Unpin and lock the beta thread.
   - [ ] Draft an announcement, include all contributors via `tools/get_release_contributors.py`
-- [ ] [Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release)
 - [ ] [Update the Changelog](https://jamulus.io/contribute/Release-Process#updating-the-changelog)
-- [ ] Tag this commit as `r3_y_z`
-  - [ ] Wait for the build to complete
+- [ ] [Tag the release version](https://jamulus.io/contribute/Release-Process#steps-for-a-specific-release):
+  - [ ] Update the version number in `Jamulus.pro` and add the release date to the Changelog header and commit.
```

Comment:
I think updating `Jamulus.pro` and `Changelog` need to be done and committed before tagging, otherwise the release will not contain the proper version and change log?

Reply:
The details seem duplicated in the referenced section.

Reply:
I'd prefer to have it here clearly as this avoids jumping:
1. Change Jamulus.pro and changelog version number
2. Commit this and tag it as rx_y_z
3. Add a new entry to the changelog and a update Jamulus.pro

Reply:
You mean everywhere we refer to `steps-for-a-specific-release`, copy the whole section in and get rid of that section, or what?  If you don't refer to that section, you won't do it right.

Reply:
We should have the lowest load as possible. So links are one step more. If we do link it, then also the "Update version number ..." stuff is unnecessary and we could just say "follow the release instructions on ..."

Reply:
There are two flows:
- tagging and releasing _any_ tagged release, be it dev, beta, rc or final release
- the additional steps necessary _only_ when tagging a final release

What I was trying to do was make that clearer.

Reply:
Latest version: http://jamuluswebsite.drealm.info/contribute/Release-Process#steps-for-a-specific-release
This probably still needs further work, to be honest.  It's referenced as "Tag the release version" here but covers mostly translations:
> #### 1. Ensure .ts files are up to date (...)
> #### 2. Notify all the translators that translation is required (...)
> #### 3. Update the .ts files returned by translators (...)
> #### 4. When all translations have been submitted and merged. (...) **<-- [This](http://jamuluswebsite.drealm.info/contribute/Release-Process#4-when-all-translations-have-been-submitted-and-merged) is where tagging starts**
>#### If this is a proper release, move the latest tag (...) *(this intended to be a sub-step of 4, I think)*

Reply:
Maybe steps 1-3 should get split into "Translation Process" and step 4 and its sub-step should be "Release Tagging Process"?

Reply:
Also, the section on updating the ChangeLog would be better placed, in the overall Release Process page, between the section about translations and the section on performing the release -- it needs to happen _before_ the final release and it never gets mentioned in the tagging section(s).

Reply:
I also notice this doc doesn't appear to reference the steps in Release Process about the translations.